examples/exchange_client/spot_exchange_rpc/8_StreamOrderbookUpdate.py

```diff
@@ -24,9 +23,9 @@ def __init__(self, market_id: str):
         self.levels = {"buys": {}, "sells": {}}
 
 
-async def load_orderbook_snapshot(client: Client, orderbook: Orderbook):
+async def load_orderbook_snapshot(async_client: AsyncClient, orderbook: Orderbook):
```

Comment:
Why don't we initialize the client outside like in the rest of the examples to simplify it? This doesn't follow convention with the rest of the methods.

Reply:
Client is initialized in main() like in the rest of the examples. We need to the async client in two functions because one streams updates while the other loads a snapshot while updates are streaming